evap/evaluation/templates/navbar.html

```diff
@@ -91,7 +91,7 @@
                         <form class="d-flex" method="post" action="{% url 'evaluation:set_lang' %}">
                             {% csrf_token %}
                             <input name="language" type="hidden" value="{{ language_code }}">
-                            <button data-bs-toggle="tooltip" data-bs-placement="bottom" title="{{ language_name }}" type="submit" onclick="setSpinnerIcon('span-set-language-{{ language_code }}')"
+                            <button data-bs-toggle="tooltip" data-bs-placement="bottom" id="button-set-language-{{ language_code }}" title="{{ language_name }}" type="submit"
```

Comment:
This comes up a lot, what do you think about `data-set-spinner-icon="span-set-language-{{ language-code }}"` and one `[data-set-spinner-icon]` selector in some JavaScript in `base.html`?